tests/cmdline/utils/test_decorators.py

```diff
@@ -67,3 +69,29 @@ def test_load_backend_if_not_loaded_with_loaded_profile(config, manager):
     # Calling the method again should keep the currently loaded profile, and not switch to the default profile
     load_backend_if_not_loaded()
     assert manager.get_profile().name == 'profile-two'
+
+
+def test_load_backend_if_not_loaded_load_once(manager, monkeypatch):
+    """Test the :meth:`aiida.cmdline.utils.decorators.load_backend_if_not_loaded` calls load profile only once."""
+    mocked = mock.Mock()
+
+    with monkeypatch.context() as context:
+        context.setattr(manager.__class__, 'get_profile_storage', mocked)
+        load_backend_if_not_loaded()
+        assert mocked.call_count == 1
+        assert not manager.profile_storage_loaded
+
+    # This is necessary, despute the previous change being in a context. Without it, subsequent tests that require
+    # the profile storage will fail.
```

Comment:
```suggestion
    # This is necessary, despite the previous change being in a context. Without it, subsequent tests that require
    # the profile storage will fail.
```

I would maybe explicitly say that we don't currently understand why it is behaving this way and from what we know it shouldn't. Otherwise it reads a bit like this is just a reminder and there was a reason that should be deducible from it.